.github/ISSUE_TEMPLATE/algorand-engineering-team-issue-template.md

```diff
@@ -8,11 +8,14 @@ assignees: ''
 
 ---
 
-## Summary
-<!-- *Describe the problem identified or the general goal of this issue* -->
+## Problem
+<!-- What is the problem that we’re trying to solve? -->
 
-## Scope/Requirements
-<!-- *What's involved in this issue? What's required to achieve the goal?* -->
+## Solution
+<!-- What is the potential/suggested Solution? Document more than one if possible. -->
```

Comment:
I know the ticket worded it this way, but maybe we should word this in a way that doesn't imply a suggested solution is necessary since a ticket creator won't always know how to solve a problem

"If you have any ideas, what is a potential/suggested solution? Document more than one if possible."

Reply:
Honestly, I agree but I think we have talk to Yaovi about it since he came up with it with the product team. I don't like implying a solution either.

Reply:
I personally prefer something like Description/Acceptance Criteria (Definition of Done) over Problem/Solution..

Reply:
@yaovi-a would you like to chime in?

Reply:
```suggestion
<!-- What is the definition of done? Do you have a potential/suggested solution? Document more than one if possible. -->
```

@yaovi-a What do you think?